src/raftstore/store/config.rs

```diff
@@ -107,13 +107,23 @@ pub struct Config {
 impl Default for Config {
     fn default() -> Config {
         let split_size = ReadableSize::mb(coprocessor::config::SPLIT_SIZE_MB);
+
+        let raft_base_tick_interval = ReadableDuration::secs(1);
```

Comment:
I am confused that you just add a metric, why change these?

Reply:
They are not changed actually, I need tick and election timeout for calculating `allowed_leader_missing_duration`, it looks clear to put them together,